lib/alaveteli_pro/webhook_endpoints.rb

```diff
@@ -0,0 +1,41 @@
+module AlaveteliPro
+  ##
+  # A class for working with the Stripe WebhookEndpoints API
+  class WebhookEndpoints
+    ##
+    # Calculates the URL of the Stripe webhook endpoint for the install
+    def self.webhook_endpoint_url
+      options = {
+        host: AlaveteliConfiguration.domain,
+        protocol: AlaveteliConfiguration.force_ssl ? 'https' : 'http'
+      }
+
+      unless AlaveteliConfiguration.include_default_locale_in_urls
```

Comment:
Doubt we ever want to `locale` in the route do we need to consider this configuration option - seems to work fine without?

Reply:
Maybe not, let's simplify it